languages/en.ini

```diff
@@ -339,6 +339,7 @@ Description = "Description"
 Desired Username = "Desired Username"
 Detailed View = "Detailed View"
 Details = "Staff View"
+Difference to top result relevance = "Difference to top result relevance"
```

Comment:
Is there a reason most of the other new strings have `explain_` prefixes but this one does not?

Reply:
This translation is not used anymore. So I deleted it. But for two other translations I also added the 'explain_' prefix.